tests/cross_functional/system_test/test_object_expiration_system.py

```
@@ -383,6 +385,15 @@ def check_if_objects_expired(mcg_obj, bucket_name, prefix=""):
         )
         wait_for_noobaa_pods_running(timeout=1200)
 
+        # Perform ceph health check after disruptions
+        try:
+            ceph_health_check(
+                namespace=config.ENV_DATA["cluster_namespace"], tries=45, delay=60
+            )
+            logger.info("Ceph health check passed after disruptions")
+        except Exception as ex:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)